.github/workflows/deploy-dev.yml

```diff
@@ -14,41 +14,21 @@ jobs:
       DEPLOY_REPO: ${{ secrets.DEV_DEPLOY_REPO }}
 
     steps:
-
-      ##########
-      ### BUILD: General SquareOne build steps
-      - name: Setup PHP, with composer and extensions
-        uses: shivammathur/setup-php@v2 #https://github.com/shivammathur/setup-php
-        with:
-          php-version: '8.2'
-        env:
-          COMPOSER_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-
       # Get Build Repository
       - name: Check out build branch
         uses: actions/checkout@v4
         with:
           path: ${{ env.BUILD_FOLDER }}
           ref: ${{github.ref}}
 
```

Comment:
This is an old deploy to WPEngine and it has been deprecated. This should be removed. Samve for staging and Production.